pkg/interop/convert/convert.go

```diff
@@ -3,15 +3,15 @@ package convert
 
 // ToInteger converts it's argument to an Integer.
 func ToInteger(v interface{}) int64 {
-	return 0
+	return v.(int64)
```

Comment:
Then do we need these methods at all? Users can just use `v.(int64)` directly.

Reply:
`v` can be any type, it is a useful wrapper, this is rather verbose:
```
x := []byte{1,2,3}
(interface{}(x)).(int64)
```